presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -123,6 +123,12 @@ Property Name                                      Description
 
 ``hive.metastore.username``                        The username Presto will use to access the Hive metastore.
 
+``hive.metastore.glue.aws-access-key``             Optional AWS access key to use as static credentials
+                                                   to connect to a Glue metastore.
+
+``hive.metastore.glue.aws-secret-key``             Optional AWS secret key to use as static credentials
+                                                   to connect to a Glue metastore.
```

Comment:
I think we should have a separate section for Glue Metastore configuration.
For example, `hive.metastore.username` and `hive.metastore.glue.aws-access-key` cannot be used together, so putting them next to each under in one table looks a bit weird.

I am aware we totally miss Glue configuration in docs.
I am therefore fine if you @pgagnon do not update docs in this PR at all.
If you could contribute Glue docs, it would be even nicer. (a separate PR)

cc @electrum

Reply:
OK. I have opened an issue #749 in order to track this.